sound/soc/sof/control.c

```diff
@@ -263,7 +263,7 @@ int snd_sof_bytes_put(struct snd_kcontrol *kcontrol,
 	pm_runtime_mark_last_busy(sdev->dev);
 	err = pm_runtime_put_autosuspend(sdev->dev);
 	if (err < 0)
-		dev_err(sdev->dev, "error: volume get failed to idle %d\n",
+		dev_err(sdev->dev, "error: bytes put failed to idle %d\n",
```

Comment:
ah crap, this conflicts with the change I just merged, so you'll have to rebase and resubmit.